modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3674,6 +3681,25 @@ private boolean _insertIntoTable(
 
 				continue;
 			}
+			else if (objectField.compareBusinessType(
```

Comment:
@pedro-oliveira446 Why are we doing it here again (inside `insertIntoTable`) since we're doing it already inside `_toObjectValues` ?